dnf/cli/commands/__init__.py

```diff
@@ -308,6 +311,12 @@ def configure(self):
             demands.sack_activation = True
             if self.opts._pkg_specs_action:
                 self.opts.pkg_specs_action = self.opts._pkg_specs_action
+            if self.opts.obsoletes:
```

Comment:
@mhatina There is an difference. The functions use different names for variable _pkg_specs_action vs.   _packages_action , and packages_action vs. pkg_specs_action. 
I can make a function but the price will be that code will be less readable.